.github/pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+# Description
+Mandatory, single line description of the pull request, written as though it was an order.
+
+Then, following a single blank line, a summary of the change and which issue is fixed, including 
+- [ ] what is the problem being solved
+- [ ] why this is the best approach
+- [ ] what are the shortcomings of this approach, if any
+It may include
+- [ ] background information, such as "Fixes #"
+- [ ] benchmark results
+- [ ] links to design documents
+- [ ] dependencies required for this change
+
+The summary may omit some of these details if this PR fixes a single ticket that includes these details. It is the reviewer's discretion. 
+
+## [Optional] Changes
+Multiple changes are not recommended, so this section should normally be omitted. Unforunately, they are sometimes unavoidable. If there are multiple logical changes, list them separately.
+
+1. `'foo'` is replaced by `'bar'`
+2. `'fizzbuzz'` is optimized for gas
+
+## Type of change
+
+Feel free to delete options that are not relevant.
+
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Chore (tech debt, improved developer experience, etc)
+- [ ] Performance improvement (either gas efficiency or computational efficiency)
+- [ ] Requires a documentation update
+
+# [Optional] How Has This Been Tested?
+
+Did you need to run manual tests to verify this change? If so, please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+# Checklist:
+
+## Code quality
+- [ ] I have written clear commit messages
+- [ ] I have performed a self-review of my own code
+- [ ] I have scoped this change as narrowly as possible
+- [ ] I have separated functional and non-functional changes
```

Comment:
Not sure I understand this.

Reply:
https://mtlynch.io/code-review-love/#6-separate-functional-and-non-functional-changes

Reply:
Got it. I bet most folks won't know what this checkbox means :) But maybe I am wrong.

Reply:
Folks were asked a couple of times to read the linked article, though I understand it might not be obvious to make the connection.

Reply:
Do you have a better description of the intended check?

Reply:
Something like: "I have separated refactor changes (whitespace, renames, etc) from logic changes"

Reply:
How about this?

```suggestion
- [ ] I have separated logic changes from refactor changes (formatting, renames, etc.)
```